### ISSUE TITLE
Feature request: Adding custom parameters to embed URL

### ISSUE THREAD

**Issue by peirix** [spicywebau/craft-embedded-assets#101]
It'd be nice to be able to append URL parameters to the embed code, to enable autoplay, muted, etc.

    {% set embeddedAsset = craft.embeddedAssets.get(asset) %}
    {{ embeddedAsset.code({ params: ['autoplay=1', 'muted=1'] }) }}

Maybe something like this?

**Comment by pvldigital** [comment]
Hi @peirix,

With release v2.0.8, i've added 2 new functions that will allow to you add additional parameters to the embedded video urls. See below for examples:
```twig
{% set vid = craft.embeddedAssets.get(entry.vid.one()) %}
{{ vid.getVideoUrl(['autoplay=1', 'controls=0', 'playsinline=1']) }}
{# 
   example output:
   https://www.youtube.com/embed/1IGS0-QF86U?feature=oembed&autoplay=1&controls=0&playsinline
#}

{{ vid.getVideoCode(['autoplay=1', 'controls=0', 'playsinline=1']) }}
{# 
   example output:
   <iframe width="1920" height="1080" src="https://www.youtube.com/embed/1IGS0-QF86U?feature=oembed&autoplay=1&controls=0&playsinline=1" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
#}

**Comment by Coysh** [comment]
Hi @pvldigital - I'm having an issue where the returned URL is adding in a ? to the URL:
`?feature=oembed?&autoplay=1&playsinline=1`

Any ideas why this is happening?

**Comment by pvldigital** [comment]
Hi @Coysh,

For the moment you can try filtering out `?` by using trim e.g `vid.getVideoCode(['autoplay=1', 'controls=0', 'playsinline=1'])|trim('?', 'left')`.

I'm not working on plugin development at the moment but @ttempleton can help you with this.

**Comment by ttempleton** [comment]
Thanks for reporting that, @Coysh -- should be fixed now in Embedded Assets 2.4.4.

**Comment by Coysh** [comment]
Thanks @ttempleton really appreciate the quick fix
